pkgs/development/libraries/nspr/default.nix

```diff
@@ -17,6 +17,13 @@ stdenv.mkDerivation {
 
   preConfigure = ''
     cd nspr
+
+    # Avoid timestamp-based impurities
+    find . -name Makefile.in -print0 | \
+      xargs -0I '{}' \
+        sed -i '{}' \
+          -e "s/shell date /shell date -d '@0' /" \
+          -e "s/^SH_NOW = .*$/SH_NOW = 0/"
```

Comment:
This will silently fail when it doesn't apply anymore. 

Could we instead craft a patch that uses [SOURCE_DATE_EPOCH](https://reproducible-builds.org/docs/source-date-epoch/)?

This should probably something upstream would be interested in, too.

Reply:
Indeed! I've changed the patch and sent it upstream, though it's still waiting for moderator approval over there.